src/test/java/org/mule/modules/kafka/automation/functional/ConsumerGroupTestCases.java

```diff
@@ -0,0 +1,8 @@
+package org.mule.modules.kafka.automation.functional;
+
+/**
+ * Created by bogdan.ilies on 21.04.2016.
```

Comment:
Your name should not appear in comments! It should be something related to MuleSoft. Please check this in all the classes.